### ISSUE TITLE
simple inference using qwen vl for fidelity, alignment

### ISSUE THREAD

**Issue by garychan22** [sjz5202/LLaVA-Reward#1]
Thanks for the great work here. And looking forward to simple inference using qwen vl for fidelity, alignment

**Comment by garychan22** [comment]
I have tried replaing the pm_path in the example with "llavareward_phi_fidelity" and the results turned out wrong in determining image quality.

```
import torch

from eval.reward_adaptor_loader import load_reward_adaptor, inference_process_phi3v, preference_compute
import os

class Args:
    pass
args = Args()

args.pm_path = "llavareward_phi_fidelity"
args.pretrain = "microsoft/Phi-3.5-vision-instruct"

args.cache_dir = None
args.ft_projector = True
args.seed = 1234
args.disable_fast_tokenizer = False

# load model
args, model, processor, tokenizer = load_reward_adaptor(
    args,
    model_type='phi3v',
    reward_config_path=os.path.join(args.pm_path, "reward_config.yaml"),
    load_tokenizer=True
)
model.to('cuda')
model.eval()

# prepare example
caption = "A young woman with long, dark hair is making a peace sign with her hand. She is wearing a black sleeveless top and has a slight smile on her face. The background features a traditional street with buildings that have signs with Chinese characters. The lighting suggests it is daytime, and the woman is standing in the foreground, with the street and buildings behind her."

img_dir_list = [
    "00.png", 
    "01.png"
]
img_inputs = inference_process_phi3v(args, processor, tokenizer, img_dir_list, caption, device='cuda')
img_inputs_c = img_inputs[0]
img_inputs_r = img_inputs[1]

# inference
with torch.no_grad():
    chosen_rewards, _ = model.custom_forward(**img_inputs_c)
    reject_rewards, _ = model.custom_forward(**img_inputs_r)

prob = preference_compute(args, chosen_rewards, reject_rewards)

# if not args.is_general_preference:
print("image0 reward:", chosen_rewards)
print("image1 reward:", reject_rewards)

print('Predict probability that image0 is better than image1:', prob) # fidelity [0.3828125]
```
image 00.png

<img width="1872" height="3075" alt="Image" src="https://github.com/user-attachments/assets/a63d07bf-ac7a-4828-92c6-bbf3fdb973db" />

image 01.png

<img width="576" height="1024" alt="Image" src="https://github.com/user-attachments/assets/ecfeed8f-4eb9-48ff-bbf6-a91433517eae" />

output:
```
Predict probability that image0 is better than image1: [0.3828125]
```

But the image 00.png is much better than image 01.png.

**Comment by sjz5202** [comment]
Hi garychan22,

Thank you for your interest! The current version of LLaVA-Reward is fine-tuned using the training set from ImageReward, where the images were generated with an early version of Stable Diffusion. This dataset has limited coverage in terms of scenarios and perspectives.

You can easily LoRA customize LLaVA-Reward for more recent image generators using the provided scripts. Please refer to the preference data format in data/sample_test as a template.

Best regards,
Shijie
